README.md

```diff
@@ -124,7 +124,8 @@ The container image includes a few lists proven to be successfully executed with
 
 * [openshift-conformance-minimal](tests-lists/openshift-conformance-minimal.txt). Based on the openshift/conformance/parallel/minimal tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance/parallel/minimal" | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
 * [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
-* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [openshift-network-stress](tests-lists/openshift_network_stress.txt).  Based on the openshift/network/stress tests suite (`openshift-tests run  openshift/network/stress | grep  -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```

Comment:
Rename the file to `openshift-network-stress`

Reply:
Done